.github/src/content/publication/abz14/arcaini2014abz2.md

```diff
@@ -2,56 +2,56 @@
 title: "Offline Model-based Testing and Runtime Monitoring of the Sensor Voting Module"
 
 authors:
-- Paolo Arcaini
-- Angelo Gargantini
-- Elvinia Riccobene
+  - Paolo Arcaini
+  - Angelo Gargantini
+  - Elvinia Riccobene
```

Comment:
@nksingh-n7 could it be that you used a editor which did automatic reformatting of all the bullet lists?

Reply:
I had used vscode for working on the website. I think the editor of vscode did automatic reformatting of all the bullet lists.

Reply:
I thought so, but it is not a problem, I'll try to comply with this format in the future with my changes and then we can keep the change set small.